layers/+spacemacs/spacemacs-navigation/README.org

```diff
@@ -21,3 +21,13 @@ This layer adds general navigation functions to all supported layers.
 - Support for =paradox= a modern emacs package manager
 - Shortcuts for restarting =emacs=
 - Shortcuts for easily switching between windows
+
+  
+* Key bindings
+
+** Info-mode
+
+| Key binding | Description                |
+|-------------+----------------------------|
+| ~J~         | Scroll up (or next node)   |
```

Comment:
I think this should rather be moved into the documentation instead.

Reply:
Ah did not realize that. So I have updated the PR.

UNRELATED: wouldn't [this new github feature](https://github.blog/changelog/2020-12-08-github-discussions-now-available-as-a-public-beta/) be handy for Spacemacs development?

Reply:
Maybe lets discuss it in a separate issue 👍